### DISCUSSION TITLE
What is the correct extension to track `.h` file changes?

### DISCUSSION THREAD

**Discussion by matu3ba** [watchexec/watchexec#368] [Questions & Help] [answered]
For example, `watchexec -e c,cpp 'make -$(nproc)'` does not work.

1. I would expect to `c` and `cpp` both track header changes, but probably they are inside another category?
2. Where is the list of file endings, for which all options in `-e` are resolved to? Maybe I did not check good enough, but I could not find it in the help page.

**Comment by passcod** [comment, marked as answer]
`-e` just tracks whatever extensions you give it, it doesn't resolve to file types. So you'd give `-e c,cpp,h` or as needed.

**Reply by matu3ba** [reply]
Thanks alot!
